etcdctlv3/README.md

```diff
@@ -0,0 +1,44 @@
+etcdctl
+========
+
+## Commands
+
+### PUT [options] \<key\> \<value\>
```

Comment:
i am not sure on this one. i changed this back and forth... value is actually required, but user can provide it in different ways.